examples/gnrc_border_router/Makefile

```diff
@@ -24,8 +36,12 @@ USEMODULE += ps
 # configure the node as a RPL DODAG root when receiving a prefix.
 #USEMODULE += gnrc_rpl
 
-# Optionally use DHCPv6 instead of UHCP
-USE_DHCPV6 ?= 0
+# When using a WiFi uplink we should use DHCPv6
+ifeq (wifi,$(UPLINK))
+  USE_DHCPV6 = 1
```

Comment:
Still make it optional? Otherwise there is no way to use UHCP with wifi, even if the user wants to

Reply:
Is there even a way to use UHCP with WiFi?